src/broker.js

```diff
@@ -46,6 +49,19 @@ module.exports = class {
         });
     }
 
+    /**
+     * Init section a.k.a queue and it's related exchanges, dead letters etc.  
+     * Assertion will be made lazy, a.k.a on consume / publish.
+     * @param section {String|Object} if section is a string, it's a ref to a named section provided at the Broker construction
+     * @param {Object} [options={}] Additional options
+     * @param {string} [options.queueName] override section.name to use the same configuration but for a different queue
+     * @param {Object} [options.sectionOverride] overrides various things at section
+     * @returns {Queue} the section
+     */
+    initQueueSection(section, options) {
```

Comment:
```suggestion
    initQueue(section, options) {
```

rename original instead of adding new